### ISSUE TITLE
Clarifying required attributes: mip_era as an example

### ISSUE THREAD

**Issue by durack1** [PCMDI/cmor#60]
@dnadeau4 @taylor13 I have found unusual behaviors when not including `mip_era` in my `CMOR3` driver file. If `mip_era` is not included in the `drive_input4MIPs_bcs/obs.json` files (as per directory structure below):
```
(cmor)durack1@oceanonly:[150219_AMIPForcingData]:[468]> ls -l */*.json
-rw-r--r-- 1 durack1 climate 2.8K Jun  3 11:48 CMOR/drive_input4MIPs_bcs.json
-rw-r--r-- 1 durack1 climate 2.6K Jun  3 11:48 CMOR/drive_input4MIPs_obs.json
(cmor)durack1@oceanonly:[150219_AMIPForcingData]:[470]> ls -l */*/Tables/*.json
-rw-r--r-- 1 durack1 climate 5.7K Jun  3 13:12 CMOR/input4MIPs-cmor-tables/Tables/input4MIPs.json
-rw-r--r-- 1 durack1 climate 2.6K Jun  2 19:55 CMOR/input4MIPs-cmor-tables/Tables/input4MIPs_CV.json
```
The following errors are triggered:
```
C Traceback:
! In function: cmor_CV_checkGblAttributes
! called from: cmor_setGblAttr
! called from: cmor_write
!

!!!!!!!!!!!!!!!!!!!!!!!!!
!
! Error: Your Control Vocabulary file specifies one or more
! required attributes.  CMOR found that the following
! attribute was not properly set.
!
! Please set attribute: "mip_era" in your input JSON file.
!
!!!!!!!!!!!!!!!!!!!!!!!!!


C Traceback:
! In function: cmor_get_cur_dataset_attribute
! called from: cmor_CV_ValidateAttribute
! called from: cmor_CV_checkGblAttributes
! called from: cmor_setGblAttr
! called from: cmor_write
!

!!!!!!!!!!!!!!!!!!!!!!!!!
!
! Error: CMOR Dataset: current dataset does not have attribute : mip_era
!
!!!!!!!!!!!!!!!!!!!!!!!!!


C Traceback:
! In function: cmor_CV_ValidateAttribute
! called from: cmor_CV_checkGblAttributes
! called from: cmor_setGblAttr
! called from: cmor_write
!

!!!!!!!!!!!!!!!!!!!!!!!!!
!
! Error: The attribute "mip_era" could not be validated.
! The current input value is "" which is not valid
! Valid values must match the regular expression:
! 	["CMIP6"  ...]
!
! Check your Control Vocabulary file "CMOR/input4MIPs-cmor-tables/Tables/input4MIPs_CV.json".
!
!
!!!!!!!!!!!!!!!!!!!!!!!!!


C Traceback:
! In function: cmor_CV_checkGblAttributes
! called from: cmor_setGblAttr
! called from: cmor_write
!

!!!!!!!!!!!!!!!!!!!!!!!!!
!
! Error: Your Control Vocabulary file specifies one or more
! required attributes.  CMOR found that the following
! attribute was not properly set.
!
! Please set attribute: "mip_era" in your input JSON file.
!
!!!!!!!!!!!!!!!!!!!!!!!!!


C Traceback:
! In function: cmor_get_cur_dataset_attribute
! called from: cmor_CV_ValidateAttribute
! called from: cmor_CV_checkGblAttributes
! called from: cmor_setGblAttr
! called from: cmor_write
!

!!!!!!!!!!!!!!!!!!!!!!!!!
!
! Error: CMOR Dataset: current dataset does not have attribute : mip_era
!
!!!!!!!!!!!!!!!!!!!!!!!!!


C Traceback:
! In function: cmor_CV_ValidateAttribute
! called from: cmor_CV_checkGblAttributes
! called from: cmor_setGblAttr
! called from: cmor_write
!

!!!!!!!!!!!!!!!!!!!!!!!!!
!
! Error: The attribute "mip_era" could not be validated.
! The current input value is "" which is not valid
! Valid values must match the regular expression:
! 	["CMIP6"  ...]
!
! Check your Control Vocabulary file "CMOR/input4MIPs-cmor-tables/Tables/input4MIPs_CV.json".
!
!
!!!!!!!!!!!!!!!!!!!!!!!!!


C Traceback:
! In function: cmor_CV_checkGblAttributes
! called from: cmor_setGblAttr
! called from: cmor_write
!

!!!!!!!!!!!!!!!!!!!!!!!!!
!
! Error: Please fix required attributes mentioned in
! the warnings above and rerun. (aborting!)
!
!
!!!!!!!!!!!!!!!!!!!!!!!!!

Traceback (most recent call last):
  File "./sanitize.py", line 301, in <module>
    cmor.write(varid,values,time_vals = d.getTime()[:],time_bnds=d.getTime().genGenericBounds())
  File "/export/durack1/anaconda2/envs/cmor/lib/python2.7/site-packages/cmor/pywrapper.py", line 696, in write
    return _cmor.write(var_id,data,type,ntimes_passed,time_vals,time_bnds,store_with)
KeyboardInterrupt
```

**Comment by durack1** [comment]
@dnadeau4 I think you have resolved this in `3.0.6`, correct?

**Comment by dnadeau4** [comment]
I fixed mip_era and return if it is not set properly. in 3.0.6.
